external-crates/move/move-compiler/tests/move_check/parser/return_in_binop.exp

```diff
@@ -14,6 +14,14 @@ warning[W09005]: dead or unreachable code
   │
   = This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
+warning[W09005]: dead or unreachable code
+  ┌─ tests/move_check/parser/return_in_binop.move:4:9
+  │
+4 │         return << 0;
+  │         ^^^^^^^^^^^ Unreachable code. This statement (and any following statements) will not be executed.
```

Comment:
This one feels a bit strange... 
I also didn't realize how much we give this `Invalid use of a divergent expression. The code following the evaluation of this expression will be dead and should be removed.` error